lib/migrations/migrations.md

```diff
@@ -0,0 +1,146 @@
+## HSD Migration
```

Comment:
I think some version of this file should be left in place in this directory, like the README in blockstore. If for no other reason, then to explain to future developers how to deploy a migration with the class. I think you've made a very elegant abstraction here, all thats needed to deploy is essentially a `check()` and `migrate()` methods.

Reply:
yep, we can rewrite this to README.